docs/pages/2019_MARVEL_Psik_MaX/sections/appendix_input_validation.md

```diff
@@ -1,3 +1,7 @@
+*These appendices consist of optional exercises, and are mentioned
```

Comment:
why not move this to the top-level index.rst?

Reply:
Because people will probably not see it...? :) As you prefer though.

Reply:
I'd rather keep them here. They are redundant in the index; here instead they serve as a reminder for people to not focus on these parts unless they have understood the rest or they have plenty of time. The appendix section titles do not say "Appendix 1.", just "1.".

Reply:
Ok, fine